lang/src/org/partiql/lang/ast/IsImplictJoinMeta.kt

```diff
@@ -17,9 +17,6 @@
  */
```

Comment:
nit: there's an empty comment block that could be deleted here

Reply:
Fixed.

Reply:
Fixed.